Added mia's advice page

miapoirier opened the pull request:
Added link on main page to new Mia P's advice page

twosavoie commented:
Hi @miapoirier,

Congrats on making your first pull request! 

I sent a message in Slack, but just in case you haven't seen it yet. This repo is shared by all Skillcrush students and adding images would mean that it would become too large quickly. So, unfortunately, in this exercise we wouldn’t be able to merge your code into to the Skillcrush repo. You can link to an image outside of the Skillcrush repo by adding its url to your CSS. You can refer how to add an image in [this lesson](https://learn.skillcrush.com/module-4/how-images-attributes-work/).

Once you've removed the img folder from the advice branch, you can add the changes by running `git add -A` (the capital A is important), commit as you usually would, and push the advice branch to GitHub. Your pull request will be updated automatically. 

Let us know if you have any questions!

Best,
Lisa

miapoirier commented:
Thank you for your quick replies Lisa - I tried what you said about putting the parent folder in the filepath and that did the trick!!😊😊😊Thanks again On 9 Nov 2024, at 18:54, Lisa Savoie ***@***.***> wrote:﻿
Hi @miapoirier,
Congrats on making your first pull request!
I sent a message in Slack, but just in case you haven't seen it yet. This repo is shared by all Skillcrush students and adding images would mean that it would become too large quickly. So, unfortunately, in this exercise we wouldn’t be able to merge your code into to the Skillcrush repo. You can link to an image outside of the Skillcrush repo by adding its url to your CSS. You can refer how to add an image in this lesson.
Once you've removed the img folder from the advice branch, you can add the changes by running git add -A (the capital A is important), commit as you usually would, and push the advice branch to GitHub. Your pull request will be updated automatically.
Let us know if you have any questions!
Best,
Lisa

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

twosavoie commented:
Hi @miapoirier,

That's awesome! If you would also like to update your pull request, while on the advice branch make sure the image folder is removed. Then add and commit the changes and push the advice branch to GitHub. Your PR will be updated automatically.

Let us know if you have any questions!

Best,
Lisa

miapoirier commented:
Hi LisaI think I did what you said. I moved my img folder to a different folder and then committed and pushed the branch. Does pushing it automatically update the pull request? I don’t have to open a new pull request? I think I need to reread the lessons as I still am confused between what a commit versus a push does…Thanks for your help!MiaOn 11 Nov 2024, at 17:12, Lisa Savoie ***@***.***> wrote:﻿
Hi @miapoirier,
You're very close! I can understand why this is so confusing since the word "advice" is used as the name of a folder and the branch you're working on.
You'll want to remove the img folder from your directory while on the advice branch. Once the img folder is removed, you can add the change by running git add -A. Then commit and push the advice branch as you normally would. Your pull request will be updated and we'll take a look!
Best,
Lisa

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

twosavoie commented:
Hi @miapoirier,

Great job taking care of the images! Well done!

It looks like the link to Gianella G's Advice has been removed. Could you add it back, then add and commit the change before pushing to GitHub? You can view the link [here](https://github.com/skillcrush/git-practice-v2/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R24) (in red.) It's fine to copy and paste it into your code.

Great questions!
- When you commit changes you're adding changes that have been staged to the directory's version history. When you run `git log` or look at the history on GitHub, you'll see a list of the commits.
- When you push to GitHub you transfer any changes committed to version history to your GitHub repo. Only committed changes are pushed. So if you make a change, but haven't committed it yet, it will not be transferred to GitHub. 
- If you have a pull request in the works, when you push the branch to GitHub (in this case the advice branch) the pull request is updated. We're able to see the changes and review them before either messaging you or merging your code into the Skillcrush repo. You can see a record of our conversations and what we see in your pull request [here](https://github.com/skillcrush/git-practice-v2/pull/8).

I hope that helps! Let us know if you have any questions!

Best,
Lisa

miapoirier commented:
Hi Lisa,

Sorry this is getting so confusing. I added that code back into index.html,
saved it, added it, committed and pushed it but I keep getting error
messages. I tried to connect to the upstream, basically following the steps
on this page:
https://learn.skillcrush.com/module-10/finishing-the-forking-workflow/, but
I still get the error below.

[image: image.png]

I think this means that I need to get the remote changes and update my
local branches with them and I thought I was doing that with the fetch, but
it's still not working. Please help.

Thank you
Mia

On Mon, 11 Nov 2024 at 23:38, Lisa Savoie ***@***.***> wrote:

> Hi @miapoirier <https://github.com/miapoirier>,
>
> Great job taking care of the images! Well done!
>
> It looks like the link to Gianella G's Advice has been removed. Could you
> add it back, then add and commit the change before pushing to GitHub? You
> can view the link here
> <https://github.com/skillcrush/git-practice-v2/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R24>
> (in red.) It's fine to copy and paste it into your code.
>
> Great questions!
>
>    - When you commit changes you're adding changes that have been staged
>    to the directory's version history. When you run git log or look at
>    the history on GitHub, you'll see a list of the commits.
>    - When you push to GitHub you transfer any changes committed to
>    version history to your GitHub repo. Only committed changes are pushed. So
>    if you make a change, but haven't committed it yet, it will not be
>    transferred to GitHub.
>    - If you have a pull request in the works, when you push the branch to
>    GitHub (in this case the advice branch) the pull request is updated. We're
>    able to see the changes and review them before either messaging you or
>    merging your code into the Skillcrush repo. You can see a record of our
>    conversations and what we see in your pull request here
>    <https://github.com/skillcrush/git-practice-v2/pull/8>.
>
> I hope that helps! Let us know if you have any questions!
>
> Best,
> Lisa
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/git-practice-v2/pull/8#issuecomment-2469292044>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLCGWPCV5PPBPIFSFAID3LT2AE5YRAVCNFSM6AAAAABRPLMOH6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINRZGI4TEMBUGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

twosavoie commented:
Hi @miapoirier,

Thanks for getting back with us! It can get confusing when we have a couple of back-and-forths in a pull request. That's perfectly normal! Removing another student's link happens sometimes when you resolve a merge conflict or accidentally when adding your own link. 

No worries! I would suggest:
-  run `git branch` to be sure you're on the **advice** branch
- copy Gianella G's link and adding it to your index.html file:  <a href="advice/gianella-guerrero-advice.html">Gianella G's Advice</a><br> 
- add and commit the change and then push the advice branch to GitHub.

The index.html file will have Gianella's link and you should be good to go! 

How does that sound? Reach out if you have any questions!

Best,
Lisa

miapoirier commented:
Hi again Lisa,

I tried that method again and got this message again:

[image: image.png]

Do I need to fetch upstream?

Thank you
Mia

On Tue, 12 Nov 2024 at 16:46, Lisa Savoie ***@***.***> wrote:

> Hi @miapoirier <https://github.com/miapoirier>,
>
> Thanks for getting back with us! It can get confusing when we have a
> couple of back-and-forths in a pull request. That's perfectly normal!
> Removing another student's link happens sometimes when you resolve a merge
> conflict or accidentally when adding your own link.
>
> No worries! I would suggest:
>
>    - run git branch to be sure you're on the *advice* branch
>    - copy Gianella G's link and adding it to your index.html file: Gianella
>    G's Advice <http://advice/gianella-guerrero-advice.html>
>    - add and commit the change and then push the advice branch to GitHub.
>
> The index.html file will have Gianella's link and you should be good to go!
>
> How does that sound? Reach out if you have any questions!
>
> Best,
> Lisa
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/git-practice-v2/pull/8#issuecomment-2471046686>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLCGWPALXVBJOOAXVO4WW6L2AIWHHAVCNFSM6AAAAABRPLMOH6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINZRGA2DMNRYGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

twosavoie commented:
Hi @miapoirier,

WooHoo!! You did a great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

Best,
Lisa
